pyanaconda/modules/localization/localization.py

```diff
@@ -200,5 +191,65 @@ def install_with_tasks(self):
         :returns: list of installation tasks
         """
         return [
-            LanguageInstallationTask(sysroot=conf.target.system_root, lang=self.language)
+            LanguageInstallationTask(
+                sysroot=conf.target.system_root,
+                lang=self.language
+            ),
+            KeyboardInstallationTask(
+                sysroot=conf.target.system_root,
+                x_layouts=self.x_layouts,
+                switch_options=self.switch_options,
+                vc_keymap=self.vc_keymap
+            )
         ]
+
+    def populate_missing_keyboard_configuration_with_task(self):
+        """Get missing keyboard configuration by conversion.
+
+        :returns: a task converting the configuration
+        """
+        if not self.x_layouts and not self.vc_keymap:
+            self.vc_keymap = DEFAULT_KEYBOARD
+
+        task = ConvertMissingKeyboardConfigurationTask(
+            x_layouts=self.x_layouts,
+            vc_keymap=self.vc_keymap,
+        )
+        task.succeeded_signal.connect(lambda: self.update_settings_from_task(task.get_result()))
+        return task
+
+    def update_settings_from_task(self, result):
```

Comment:
I think that this method could be private.

Reply:
https://github.com/rhinstaller/anaconda/pull/2217/commits/ca85dad74ca838176b8f128e02dec24c0b2da0e3